src/backend/launcher.ts

```diff
@@ -394,6 +386,45 @@ function setupWrappers(
   return wrappers.filter((n) => n)
 }
 
+/**
+ * Checks if the game's selected Wine version exists
+ * @param game The game to check the Wine version of
```

Comment:
Shouldn't this be `wineVersion` instead of `game`?

Reply:
yep!

Reply:
```suggestion
 * @param wineVersion an object of type WineInstallation with binary path and name to check
```

Reply:
```suggestion
 * @param wineVersion an object of type WineInstallation with binary path and name to check
```